### PR TITLE
Feature/#294 tables drag and drop not working in tablet

### PR DESCRIPTION
**Pull request by migalko94**
Close#294

### CODE REVIEW

File: src/pods/canvas/canvas.pod.tsx [hunk 1]
```diff
@@ -80,14 +79,15 @@ export const CanvasPod: React.FC = () => {
...
-  };
+  // const handleScroll = () => {
+  //   if (containerRef.current) {
+  // setPosition comes from the useCanvasViewSettingsContext
+  //     setScrollPosition({
+  //       x: containerRef.current.scrollLeft,
+  //       y: containerRef.current.scrollTop,
+  //     });
+  //   }
+  // };
```
Comment (brauliodiez) [inline src/pods/canvas/canvas.pod.tsx:90]: Should this code be removed if it's commented? Or explain why is commented

File: src/pods/canvas/canvas.pod.tsx [hunk 2]
```diff
@@ -145,7 +145,7 @@ export const CanvasPod: React.FC = () => {
     <div
       className={classes.container}
       ref={containerRef}
-      onScroll={handleScroll}
+      // onScroll={handleScroll}
```
Comment (brauliodiez) [inline src/pods/canvas/canvas.pod.tsx:148]: Same here

File: src/pods/canvas/canvas.pod.tsx (hunk 1 above)
Comment (deletidev) [reply src/pods/canvas/canvas.pod.tsx]: It's fixed. I forgot what the function does.

File: src/pods/canvas/canvas.pod.tsx (hunk 2 above)
Comment (deletidev) [reply src/pods/canvas/canvas.pod.tsx]: It's fixed.

File: src/pods/canvas/canvas.pod.tsx (hunk 1 above)
Comment (brauliodiez) [reply src/pods/canvas/canvas.pod.tsx:90]: Now the add collection and place the table where the scroll is is broken, can you take a look?

Comment (deletidev) [reply src/pods/canvas/canvas.pod.tsx]: Fixed, something happened with my last commit.

**Pull request merged**